crates/api_common/src/site.rs

```diff
@@ -152,6 +152,7 @@ pub struct CreateSite {
   pub captcha_difficulty: Option<String>,
   pub allowed_instances: Option<Vec<String>>,
   pub blocked_instances: Option<Vec<String>>,
+  pub taglines: Option<Vec<String>>,
   pub auth: Sensitive<String>,
```

Comment:
Not really related to this PR, but does it really make sense to include all these fields here? Afaik lemmy-ui doesnt allow setting most of them during setup, so there is no reason to have all this additional code here and in the CreateSite API handler. I would only include the fields which can actually be set by lemmy-ui site creation.

Reply:
Good idea; removed from CreateSite

Reply:
These can all be set during the site creation as well, but it up to different UIs to determine which ones they want to show. 

Its good to have the EditSite fields pretty much match the CreateSite fields, otherwise people will ask things like: "why can I set `allow nsfw` when I create a site, but not set `admin_only_community_creation`?

Reply:
lol okay ill re-add it

Reply:
> These can all be set during the site creation as well, but it up to different UIs to determine which ones they want to show.

This is a moot point as long as no different UI exists which supports this. If such a project is created and wants to support these settings during site creation, we would have plenty of time to add them. So for now its just dead code.

Anyway this can be done in another PR.